packages/react-examples/src/react-charting/GaugeChart/GaugeChartPage.tsx

```diff
@@ -47,36 +47,68 @@ export class GaugeChartPage extends React.Component<IComponentDemoPageProps, {}>
+                <code>segments</code>: Use this required prop to divide the gauge into colored sections. The segments
+                can have fixed sizes or vary with the chart value to create a sweeping effect. Negative segment sizes
+                are treated as 0.
               </li>
               <li>
-                Set the <code>minValue</code> prop if the minimum value of the gauge is different than 0. A placeholder
-                segment will be rendered if the <code>maxValue</code> prop is greater than the total size of the
-                segments.
+                <code>minValue</code>: Use this prop if the minimum value of the gauge is different from 0.
               </li>
               <li>
-                To render an additional text below the chartValue, set the <code>sublabel</code> prop.
+                <code>maxValue</code>: Use this prop to render a placeholder segment when the difference between the max
+                and min values is larger than the total size of the segments. If the difference is smaller, the max
+                value will be adjusted so that the total size of the segments matches the difference.
               </li>
               <li>
-                To hide the minimum and maximum values of the gauge, set the <code>hideMinMax</code> prop.
+                <code>sublabel</code>: Use this prop to render additional text below the chart value.
               </li>
               <li>
-                The chartValue prop is rendered as a percentage by default. Set the <code>chartVaueFormat</code> prop to
-                'fraction' or a formatter function.
+                <code>hideMinMax</code>: Set this prop to true to hide the min and max values of the gauge.
+              </li>
+              <li>
+                <code>chartValueFormat</code>: This prop controls how the chart value is displayed. Set it to one of the
+                following options:
+                <ul>
+                  <li>A custom formatter function that returns a string representing the chart value.</li>
+                  <li>
+                    <code>GaugeValueFormat.Fraction</code>: Renders the chart value as a fraction.
+                  </li>
+                  <li>
+                    <code>GaugeValueFormat.Percentage</code>: Renders the chart value as a percentage. This is the
+                    default format.
+                  </li>
+                </ul>
+                Note: If the min value is non-zero and no formatter function is provided, the chart value will be
+                rendered as a number.
+              </li>
+              <li>
+                <code>variant</code>: This prop determines the presentation style of the gauge chart. Set it to one of
+                the following options:
+                <ul>
+                  <li>
+                    <code>GaugeChartVariant.SingleSegment</code>: Displays and announces the segment sizes as
```

Comment:
Clarify this so it means - This is meant to represent a single metric value out of a target goal (n/M).

Reply:
done